libs/base/src/lib/components/core/ExplorerLink.tsx

```diff
@@ -3,18 +3,20 @@ import React, { FC } from 'react'
 import { truncateAddress } from '@apps/formatters'
 import { ExternalLink } from '@apps/dumb-components'
 
-import { useGetExplorerUrl } from '../../context/NetworkProvider'
+import { ChainIds, getNetwork, useNetwork } from '../../context/NetworkProvider'
 
 export const ExplorerLink: FC<{
   data: string
   type?: 'transaction' | 'account' | 'address' | 'token'
   truncate?: boolean
   showData?: boolean
   className?: string
-}> = ({ children, type = 'address', data, showData, truncate = true, className }) => {
-  const getExplorerUrl = useGetExplorerUrl()
+  chainId?: ChainIds
```

Comment:
You can now ask for an explorer link on a chain you're not connected to (useful for dials table)